result.go

```diff
@@ -381,17 +381,42 @@ func (q Query) ToMapBy(
 	res.Elem().Set(m)
 }
 
-// ToSlice iterates over a collection and populates the result slice with elements.
+// ToSlice iterates over a collection, copy to result slice the collection elements,
```

Comment:
> "iterates over a collection" ––do we mean "slice"? what are some other possible collections here?

@ahmetalpbalkan I don't know, I'm just following the current documentation terminology like [here](../master/result.go#L333) , [here](../master/result.go#L345)  and [here](../master/result.go#L360) 

> "copy to result slice the collection elements" ––sounds like a grammatical mistake here?

Sorry man, English is not my first language. Will be amazing have more help with documentation.